lib/validate.js

```diff
@@ -69,6 +69,7 @@ export function advPostSchemaMembers (client) {
         },
         message: `must be divisble be ${BOOST_MIN}`
       }),
+    // XXX this lets you forward to youself (it's financially equivalent but it should be disallowed)
```

Comment:
Ah yes, I thought about this at one point but forgot about it. I guess this could be validated at the API tier where we have access to `me`?

Reply:
Yes, I think we'd want to pass it like we pass the client ... which is currently a pretty hacky dep injection.